msbuild/Xamarin.iOS.Tasks.Core/Xamarin.TVOS.AppExtension.CSharp.targets

```diff
@@ -22,7 +22,9 @@ Copyright (C) 2014-2016 Xamarin. All rights reserved.
 		<TargetFrameworkIdentifier Condition="'$(TargetFrameworkIdentifier)' == ''">Xamarin.TVOS</TargetFrameworkIdentifier>
 		<TargetFrameworkVersion Condition="'$(TargetFrameworkVersion)' == ''">v1.0</TargetFrameworkVersion>
 
-		<DefineConstants>__UNIFIED__;__MOBILE__;__TVOS__;$(DefineConstants)</DefineConstants>
+		<DefineConstants Condition="!$(DefineConstants.Contains('__UNIFIED__'))">__UNIFIED__;$(DefineConstants)</DefineConstants>
+		<DefineConstants Condition="!$(DefineConstants.Contains('__MOBILE__'))">__MOBILE__;$(DefineConstants)</DefineConstants>
+		<DefineConstants Condition="!$(DefineConstants.Contains('__TVOS__'))">__TVOS__;$(DefineConstants)</DefineConstants>
```

Comment:
It's probably more complicated than this because what if I define `__SOMETHING__UNIFIED__`? or `__UNIFIED__SOMETHING__`?

Probably not *likely*, but it it's not quite that trivial.